prow/cmd/sub/README.md

```diff
@@ -75,3 +76,67 @@ This will find and start the periodic job ```my-periodic-job```, and add / overw
 annotations and envs to the Prow job. The ```prow.k8s.io/pubsub.*``` annotations are
 used to publish job status.
 
+#### Presubmit Prow Jobs
+
+Triggering presubmit job is similar to periodic jobs. Two things to change:
+
+- instead of an attributes with key ```prow.k8s.io/pubsub.EventType``` and value
+```prow.k8s.io/pubsub.PeriodicProwJobEvent```, replace the value with ```prow.k8s.io/pubsub.PresubmitProwJobEvent```
+- requires setting `refs` instructing presubmit jobs how to clone source code:
+
+```json
+{
+  // Common fields as above
+  "name":"my-presubmit-job",
+  "envs":{...},
+  "labels":{...},
+  "annotations":{...},
+
+  "refs":{
+    "org": "org-a",
+    "repo": "repo-b",
+    "base_ref": "main",
+    "base_sha": "abc123",
+    "pulls": [
+      {
+        "sha": "def456"
+      }
+    ]
+  }
+}
+```
+
+This will start presubmit job ```my-presubmit-job```, clones source code like pull requests
+defined under ```pulls```, which merges to ```base_ref``` at ```base_sha```.
+
+(There are more fields can be supplied, see [full documentation](https://github.com/kubernetes/test-infra/blob/18678b3b8f4bc7c51475f41964927ff7e635f3b9/prow/apis/prowjobs/v1/types.go#L883). For example, if you want the job to be reported on the PR, add ```number``` field right next to ```sha```)
```

Comment:
This seems a bit misleading. I'm pretty sure reporting will happen whether or not the PR number is supplied. IIRC the PR number is used for things like constructing links, aggregation in Tide, and front end job filtering, but ProwJob results are reported to GitHub based on the commit SHA, not the PR number.